perf/lmbench.py.data/0001-Fix-build-issues-with-lmbench.patch

```diff
@@ -42,7 +42,7 @@ diff --git a/scripts/config-run b/scripts/config-run
 index b8c17d5..9958a31 100755
 --- a/scripts/config-run
 +++ b/scripts/config-run
```

Comment:
@nasastry what changed here?

Reply:
I didn't change any thing over there ... seems that's the syntax.
```
diff --git a/perf/lmbench.py.data/0001-Fix-build-issues-with-lmbench.patch b/perf/lmbench.py.data/0001-Fix-build-issues-with-lmbench.patch
index e9778b1b..bbe699ae 100644
--- a/perf/lmbench.py.data/0001-Fix-build-issues-with-lmbench.patch
+++ b/perf/lmbench.py.data/0001-Fix-build-issues-with-lmbench.patch
@@ -42,7 +42,7 @@ diff --git a/scripts/config-run b/scripts/config-run
 index b8c17d5..9958a31 100755
 --- a/scripts/config-run
 +++ b/scripts/config-run
-@@ -115,9 +115,12 @@ three attendent child processes sending data down the pipes and
+@@ -115,9 +115,12 @@ three attendant child processes sending data down the pipes and
  three benchmark processes reading data and doing the measurements.

  EOF
```
And other example is:
```
diff --git a/perf/perf_24x7_all_events.py b/perf/perf_24x7_all_events.py
index 2d4b79f5..7b6df106 100755
--- a/perf/perf_24x7_all_events.py
+++ b/perf/perf_24x7_all_events.py
@@ -67,7 +67,7 @@ class hv_24x7_all_events(Test):
         event_sysfs = "/sys/bus/event_source/devices/hv_24x7"

         # Check if this is a guest
-        # 24x7 is not suported on guest
+        # 24x7 is not supported on guest
         if "emulated by" in cpu._get_info():
             self.cancel("This test is not supported on guest")
```